src/x_file.c

```diff
@@ -1411,7 +1502,8 @@ void x_file_setup(void)
         gensym("close"), 0);
     class_addmethod(file_handle_class, (t_method)file_handle_seek,
         gensym("seek"), A_GIMME, 0);
-    class_addfloat(file_handle_class, file_handle_float);
+    class_addmethod(file_handle_class, (t_method)file_handle_set,
+        gensym("set"), A_DEFSYMBOL, 0);
```

Comment:
Maybe it would be more idiomatic to set the symbol via a second inlet to match the syntax of the `[text]` and `[array]` objects.

Reply:
actually there *is* a 2nd inlet to set it.

Reply:
Ah, sorry, now I see it.